test/features/forms/MainDonationDonationReceipt_UpgradeToYearlyButton.ts

```diff
@@ -0,0 +1,23 @@
+import { VueWrapper } from '@vue/test-utils';
+import { donationFormFeatures as mainDonationFormFeatures } from '@test/features/forms/MainDonation_UpgradeToYearlyButton';
+import { expect } from 'vitest';
+
+export const donationFormFeatures: Record<string, ( wrapper: VueWrapper<any> ) => Promise<any>> = {
+	expectMainDonationFormSubmitsWhenSofortIsSelected: mainDonationFormFeatures.expectMainDonationFormSubmitsWhenSofortIsSelected,
+	expectMainDonationFormSubmitsWhenYearlyIsSelected: mainDonationFormFeatures.expectMainDonationFormSubmitsWhenYearlyIsSelected,
+	expectMainDonationFormSubmitsWhenAmountIsGreaterThanMaxAmount: mainDonationFormFeatures.expectMainDonationFormSubmitsWhenAmountIsGreaterThanMaxAmount,
+	expectMainDonationFormGoesToUpgrade: mainDonationFormFeatures.expectMainDonationFormGoesToUpgrade,
+	expectUpgradeToYearlyFormSubmitsUpgrade: mainDonationFormFeatures.expectUpgradeToYearlyFormSubmitsUpgrade,
+	expectUpgradeToYearlyFormSubmitsDontUpgrade: mainDonationFormFeatures.expectUpgradeToYearlyFormSubmitsDontUpgrade,
+	expectUpgradeToYearlyFormGoesToMainDonation: mainDonationFormFeatures.expectUpgradeToYearlyFormGoesToMainDonation,
```

Comment:
Did you want to pick and choose specific features? Otherwise you could do `...mainDonationFormFeatures` instead.

Reply:
Oh yeah, didn't think of that.